CONTRIBUTING.md

```diff
@@ -490,3 +490,30 @@ We have a few procedures we follow. These are documented here:
   
   </p>
   </details>
+
+### Set up a Docker Builder
```

Comment:
Can we wrap this in a `<details>` block as the other ones above or move this to the `Release` part of the contribution guide? (making who we are adressing in this clearer)